recipe/build.sh

```diff
@@ -42,3 +42,5 @@ for i in `ls`; do
         done
     fi
 done
+
+check-glibc "$PREFIX"/lib*/*.so.* "$PREFIX"/bin/* "$PREFIX"/targets/*/lib*/*.so.* "$PREFIX"/targets/*/bin/*
```

Comment:
We could either go ahead and run this as-is (it is harmless and maybe useful in the future)

Or we could drop it like we did elsewhere: https://github.com/conda-forge/cuda-profiler-api-feedstock/pull/17/commits/68b85d897b4db185d28b2bdb0e50166e8961d8eb

```suggestion
```